### DISCUSSION TITLE
FX Editing

### DISCUSSION THREAD

**Discussion by Maxklui** [bangcorrupt/hacktribe#207] [General]
Hello

I have a general question about FX editing. What will it look like in the end when this update comes? Because I need, among other things, reverb as an insert FX, as an example now. And what will that look like, so that anyone can do it, even if they don't have any Python knowledge.

Thanks in advance

**Comment by bangcorrupt** [comment]
Hopefully we will be able to edit the parameters on the device, maybe only via MIDI.  It won't require Python programming.
